superset-frontend/src/explore/components/ExploreChartPanel.jsx

```diff
@@ -109,6 +126,16 @@ const Styles = styled.div`
   }
 `;
 
+const requiredFieldsMissingWarning = isFeatureEnabled(
+  FeatureFlag.ENABLE_EXPLORE_DRAG_AND_DROP,
+)
+  ? t(
+      'Drag and drop values into highlighted field(s) in the control panel. Then run the query by clicking on the "Update chart" button.',
+    )
+  : t(
+      'Select values in highlighted field(s) in the control panel. Then run the query by clicking on the "Update chart" button.',
+    );
```

Comment:
As this is identical to the text in `src/components/Chart/Chart.tsx`, we should perhaps consider placing these strings in a `constants.ts` to ensure they stay the same (otherwise someone might only update one but not the other).

Reply:
Not exactly identical - the other one says "Create chart", this one "Update chart". Though we should probably make it more generic, like move to a function with argument `isCreating` or `isUpdating`